python/pdstools/ih/Plots.py

```diff
@@ -8,4 +14,130 @@
 
 class Plots(LazyNamespace):
     def __init__(self, ih: "IH_Class"):
+        super().__init__()
         self.ih = ih
+
+    def experiment_gauges(
+        self,
+        experiment_field: str,
+        by: Optional[str] = "Channel",
+        positive_labels: Optional[List[str]] = None,
+        negative_labels: Optional[List[str]] = None,
+        reference_values: Optional[Dict] = None,
```

Comment:
Worth type hinting the internal structure of the Dict type? Might make the API easier to use